site/de/spec/glossar.md

```diff
@@ -0,0 +1,360 @@
+---
+layout: page
+lang: de
+lang-ref: glossary.md
+toc: true
+title: OCR-D Glossar
+---
+
+# OCR-D Glossar
+
+> Glossar von Begriffen aus dem Bereich der Bildverarbeitung/OCR und deren Verwendung im Rahmen von OCR-D
+
+Dieser Abschnitt ist nicht-normativ.
```

Comment:
haha, we should probably drop this here and in the english variant, sounds so self-important. And it is kinda normative.